### PR TITLE
chore(wrapper): rhidp-4257 add topology wrapper

### PR DESCRIPTION
**Pull request by ciiay**
## Description

Added wrappers for backstage-community/plugin-topology and backstage-community/plugin-topology-common. Verified it exports the topology plugin statically and dynamically (with changes in BCP PR) with no issues.

## Which issue(s) does this PR fix

- Migrating topology plugin from backstage-plugins repo to BCP
- Blocked by: https://github.com/backstage/community-plugins/pull/1470
- ~Blocked by: https://github.com/backstage/community-plugins/pull/1723~
- Blocked by: https://github.com/backstage/community-plugins/pull/1827

## PR acceptance criteria

Please make sure that the following steps are complete:

- [x] GitHub Actions are completed and successful
- [x] Unit Tests are updated and passing
- [x] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

## How to test changes / Special notes to the reviewer
1. Go to `dynamic-plugins/wrappers/backstage-community-plugin-topology/` directory
2. Run `yarn install` and `yarn export-dynamic` to see it runs both scripts with no error

### CODE REVIEW

**Comment by invincibleJai** [comment]
changes looks good, need to try it once

File: dynamic-plugins/wrappers/backstage-community-plugin-topology/package.json [hunk 1]
```diff
@@ -1,34 +1,45 @@
 {
-  "name": "janus-idp-backstage-plugin-topology",
-  "version": "1.30.0",
+  "name": "backstage-community-plugin-topology",
+  "version": "1.29.4",
```
Comment (christoph-jerolimov) [inline dynamic-plugins/wrappers/backstage-community-plugin-topology/package.json:3]: This downgrade is right, but we need to increase the backstage version to 1.31 or something similar and check again that all updates are migrated!

* https://www.npmjs.com/package/@janus-idp/backstage-plugin-topology?activeTab=versions is currently 1.30.1
* https://www.npmjs.com/package/@backstage-community/plugin-topology?activeTab=versions is currently 1.29.4

**Review by christoph-jerolimov — APPROVED** [review]
Verified on a cluster

/lgtm

File: dynamic-plugins/wrappers/backstage-community-plugin-topology/package.json (hunk 1 above)
Comment (ciiay) [reply dynamic-plugins/wrappers/backstage-community-plugin-topology/package.json:3]: Hi @christoph-jerolimov , do you mean manually increase the @bacstage-community/plugin-topology version to 1.31? This part is handled by workflow, is it okay to manually change it?

**Pull request merged**